src/optiondefs.h

```diff
@@ -1147,12 +1147,12 @@ static struct vimoption options[] =
 			    (char_u *)&p_gp, PV_GP, NULL, NULL,
 			    {
 # if defined(MSWIN)
-			    // may be changed to "grep -n" in os_win32.c
+			    // may be changed to "grep -In" in os_win32.c
```

Comment:
If you update this comment, it should be better to update this part:
https://github.com/vim/vim/blob/9a39483adb418e37c672000a58792c0f0e8aa662/src/os_win32.c#L3044-L3050

BTW, does the `-I` option work on non-GNU grep?

Reply:
My macOS manual includes `-I` with the same meaning, but I'd rather not rely on non-portable defaults. (That is, for the \*nixes, stick with POSIX unless we can detect we are running somewhere that supports the other flags.) POSIX does not specify `-I` or `-H`, so both are extensions.

(`-I` _may_ be common enough in the wild to justify on newer Vim versions; I am not an expert on what other implementations of `grep` are floating around.)

Reply:
But we are not using `grep -H`. Not sure why this matters here. FWIW: I checked OpenBSD and NetBSD man pages. It includes `-I` for release `2.7` (from 2000` and NetBSD, which mentions `-I` for NetBSD 2 (released 2004). For FreeBSD I wasn't able to find man pages for older versions.

So might be okay, but given the recent backlash on changing defaults, I am not sure 🤷‍♂️